lib/identity_config.rb

```diff
@@ -395,6 +395,8 @@ def self.store
     config.add(:sign_in_recaptcha_percent_tested, type: :integer)
     config.add(:sign_in_recaptcha_score_threshold, type: :float)
     config.add(:skip_encryption_allowed_list, type: :json)
+    config.add(:socure_document_request_endpoint, type: :string)
+    config.add(:socure_id_plus_secret_key, type: :string)
```

Comment:
as per our paring session, let's reuse `socure_idplus_api_key` which already exists instead of adding this duplicate key anew?
```suggestion
```